Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -94,4 +115,77 @@ gdjs.LightRuntimeObjectPixiRenderer.prototype.updateVertexBuffer = function () {
         this._object.y,
       ])
     );
+//  }
+};
+
+gdjs.LightRuntimeObjectPixiRenderer.prototype.raycastTest = function () {
+  var result = [];
+  var center = [this._object.x + this._radius, this._object.y + this._radius];
```

Comment:
See other comments, I recommend we use this._object.x and this._object.y as the center coordinates.

Reply:
Also, unless you need to pass this as a point to a function, avoid allocating an array in memory and just directly use `this._object.x` and `this._object.y`.
If you want to name them as center, do `var centerX = this._object.x;` and `var centerY = this._object.y;`.

Not a huge deal, but avoiding allocating arrays and objects can help reducing the pressure on the garbage collector.